src/app/tests/suites/TestModeSelectCluster.yaml

```diff
@@ -112,6 +112,14 @@ tests:
       response:
           value: currentModeBeforeToggle
 
+    - label: "Change to Unsupported OnMode"
```

Comment:
This test doesn't seem to actually exercise what this PR is implementing.  Can we please exercise it?

Reply:
Will do